lock/src/main/java/com/auth0/android/lock/LockActivity.java

```diff
@@ -359,7 +418,7 @@ public void onFailure(final Auth0Exception error) {
             handler.post(new Runnable() {
                 @Override
                 public void run() {
-                    progress.showResult(error.getMessage());
+                    setResultMessage(R.string.com_auth0_lock_result_message_generic_error, false);
```

Comment:
If we ever need to use a different error message we need to track all of these usages. Just extract to a method